js/hangman/game.js

```diff
@@ -107,6 +107,8 @@ function start() {
  * Guess button handler
  */
 function guess() {
+    $("#letter").select();
```

Comment:
`$('#letter')` is already stored at `GAME_UI_COMPONENTS` constant so, the right way to get it is by doing: `ui.letter.select();`. Also, this must go in line 121, before assigning `choice` variable.